### DISCUSSION TITLE
Welcome to Half-Life-Resurgence Discussions!

### DISCUSSION THREAD

**Discussion by DrVrej** [VJ-HLR-Developers/Half-Life-Resurgence#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to ask quick questions, rather then posting it in issues 👍

**Comment by EzzatBoukhary** [comment]
Hello. Does it require mounting of Half-Life: Source or any other game? Or just the VJ Base addon?

Also, I don't know if this is intentional or not but when I tried downloading the code folder it says that the archive is corrupt.

**Comment by DrVrej** [comment]
> Hello. Does it require mounting of Half-Life: Source or any other game? Or just the VJ Base addon?
> 
> Also, I don't know if this is intentional or not but when I tried downloading the code folder it says that the archive is corrupt.

Only VJ Base is required. If you are having trouble downloading, I recommend to wait until it's released on Workshop.

**Comment by VINXANDER4998** [comment]
Greetings, DrVrej. I have a problem going on with my mod HLR. I don't get the HLR snpcs. Do i have to download another mod because i've downloaded VJ Base

**Comment by jolopapina** [comment]
Will you add rappel ability to combine soldiers?
